src/App.js

```diff
@@ -1,7 +1,17 @@
+import LeftNav from "frontend/components/leftNav.jsx";
+import RightNav from "frontend/components/rightNav.jsx";
+import Homepage from "frontend/pages/homepage/homepage.jsx";
+import { Routes, Route } from "react-router-dom";
+
 function App() {
   return (
-    <div className="App">
-      <h1>Spectrum</h1>
+    <div className="App flex justify-between w-screen">
+      <LeftNav />
+      <Routes>
+        <Route path="/" element={<Homepage />} />
+        <Route path="*" element={<div style={{ display: "flex", justifyContent: "center" }}><h1 style={{ fontSize: "5rem" }}>Page Not Found!</h1></div>} />

```

Comment:
any specific reason why inline css is used in here?
you can create a page not found component and use that instead.

Reply:
Saves the app bloat and keeping it simple